euphonic/cli/intensity_map.py

```diff
@@ -34,15 +34,17 @@ def main(params: Optional[list[str]] = None) -> None:
     if (not frequencies_only
         and not isinstance(data, (QpointPhononModes, ForceConstants))
     ):
-        raise TypeError('Eigenvectors are required to use '
-                        '"--weighting coherent" option')
+        msg = 'Eigenvectors are required to use "--weighting coherent" option'
+        raise TypeError(msg)
     if (args.weighting.lower() == 'coherent'
         and args.temperature is not None
         and not isinstance(data, ForceConstants)
     ):
-        raise TypeError('Force constants data is required to generate '
-                        'the Debye-Waller factor. Leave "--temperature" '
-                        'unset if plotting precalculated phonon modes.')
+        msg = dedent_and_fill("""\
```

Comment:
By the way, is there a reason to not strip newlines in `dedent_and_fill` so you don't need the `"""\`?

Reply:
I guess the function could handle it, yes!

The main problem isn't that the newline exists, by the way, but that it breaks the indentation detection.